teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -263,6 +448,27 @@ interface Revision {
     val vcsRootInstance: VcsRootInstance
 }
 
+enum class TestStatus {
+    SUCCESSFUL,
+    IGNORED,
+    FAILED,
+
+    UNKNOWN,
+}
+
+interface TestOccurrence {
+    val name : String
+    val status: TestStatus
+    val duration: Long
```

Comment:
`duration` is in milliseconds, right? It makes sense to mention this in the name or a KDoc.